Sources/XcodeProj/Utils/BuildSettingsProvider.swift

```diff
@@ -373,10 +373,6 @@ public class BuildSettingsProvider {
 
     private static func targetSwiftSettings(platform: Platform, product: Product) -> BuildSettings {
```

Comment:
I believe the method can be removed completely now

Reply:
@fortmarek I agree. Removed it!